src/ZODB/interfaces.py

```diff
@@ -318,6 +319,14 @@ def invalidate(transaction_id, oids):
 
         The oids argument is an iterable of object identifiers.
 
+        Unless cache needs to be completely cleared via invalidateCache
+        event, wrapped storage calls invalidate for all transactions in
+        the order as they are committed. For every transaction full set
```

Comment:
"the order they are committed. For every transaction __the__ full set"

Reply:
ok